addons/board/static/src/add_to_board/add_to_board.js

```diff
@@ -34,10 +34,10 @@ export class AddToBoard extends Component {
     //---------------------------------------------------------------------
 
     async addToBoard() {
-        const { domain, globalContext } = this.env.searchModel;
+        const { domain, _context } = this.env.searchModel;
```

Comment:
I don't get it. _context (or better context) is a mix of searchModel._getContext() and searchModel.globalContext but context from next line is also a mix of searchModel._getContext() and searchModel.localContext. How is it supposed to fix the problem?

Reply:
Could you give more information in your commit message?